actionpack/test/controller/http_token_authentication_test.rb

```diff
@@ -80,13 +80,20 @@ def authenticate_long_credentials
   end
 
   test "authentication request with badly formatted header" do
-    @request.env['HTTP_AUTHORIZATION'] = "Token foobar"
+    @request.env['HTTP_AUTHORIZATION'] = 'Token token$"lifo"'
```

Comment:
@arthurnn the token had a valid format but not valid content, so i used a valid token but formatted badly (using the `$`)